pkgs/development/python-modules/chime/default.nix

```
@@ -0,0 +1,35 @@
+{
+  alsa-utils,
+  buildPythonPackage,
+  fetchFromGitHub,
+  lib,
+  poetry-core,
+  stdenv,
+  unstableGitUpdater,
+}:
+
+buildPythonPackage {
+  pname = "chime";
+  version = "0-unstable-2023-03-06";
```

```suggestion
  version = "0.7-unstable-2023-03-06";
```
Last "release" was 0.7